docker-compose.yml

```diff
@@ -93,6 +93,7 @@ services:
       - FLASK_ENV=development
       - AUTHENTICATOR_HOST=http://localhost:4004
       - DATA_STORE_API_HOST=http://data-store:8080
+      - NOTIFY_API_KEY=${NOTIFY_API_KEY_SECRET:?err}
```

Comment:
I think this setting makes it mandatory? If we don't want that we could comment it out?

Reply:
Do we want to set the env var to enable notifications in that case?